docs/monitoring/metrics.md

```diff
@@ -896,6 +896,28 @@ Thus, in order to infer the metric identifier:
   </tbody>
 </table>
 
+#### Connector:
+<table class="table table-bordered">
+  <thead>
+    <tr>
+      <th class="text-left" style="width: 20%">Scope</th>
+      <th class="text-left" style="width: 30%">Metrics</th>
+      <th class="text-left" style="width: 50%">Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th rowspan="1">Slot/Consumer</th>
```

Comment:
This documentation is inconsistent with the rest.

Scope should be "Operator", and you should add an additional "Infix" column which contains the names of the metric groups you are creating, concatenated with a period.